<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusView2</class>
 <widget class="QWidget" name="StatusView2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>968</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QChartView" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>120</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>470</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/箭头.png</normaloff>image/箭头.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::DelayedPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWebEngineView" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>540</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="VlcWidgetVideo" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>40</y>
     <width>841</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>290</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="PanelWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>690</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>920</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </widget>
  <zorder>widget_3</zorder>
  <zorder>widget</zorder>
  <zorder>widget_4</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_5</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelWidget</class>
   <extends>QWidget</extends>
   <header>panelwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
